server/search/github/github.test.js

```diff
@@ -0,0 +1,60 @@
+import chai from 'chai';
+const expect = chai.expect;
+const Promise = require('bluebird')
+const github = require('./request')
+const formatter = require('./formatter')
+
+
+describe('GitHub API:', () => {
+  let result
+
+  before(() => { result = github('sequelize error') })
```

Comment:
You can keep this as a smoke test for the API to run manually at times to check it is still ok… but should not be part of the normal testing suite.

For anything that USES GH data, use a mock.